### PR TITLE
Add preliminary schedule

### PR DESCRIPTION
**Pull request by Hoverbear**

### CODE REVIEW

**Comment by gandro** [comment]
Looks good to me. I like the ordering, the times seem to add up.

File: schedule.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+| How I Convinced the World's Largest Package Manager to Use Rust, and So Can You! | Ashley Williams | 11:30-12:00 |
+| Writing Idiomatic Libraries in Rust | Pascal Hertleif | 12:10-12:40 |
+| Lunch | | 12:40-1:40 |
+| Making a game in Rust | lislis | 1:40-1:55 |
+| GStreamer & Rust – A perfect match | Sebastian Dröge & Luis de Bethencourt | 2:05-2:35 |
+| Sōzu, a hot reconfigurable reverse HTTP proxy | Geoffroy Couprie | 2:45-3:25 |
+| Break | | 3:25-3:40 |
+| Interaction with real-world JavaScript from compiled Rust | Ingvar Stepanyan | 3:40 - 4:10  |
+| vagga -- containerization without daemons | Paul Colomiets | 4:20 - 5:00 |
+| TODO: Automation in Rust | Emily Dunham | 5:15-6:00 |
```
Comment (gnunicorn) [inline schedule.md]: Did you include/anticipate the surprise of our main sponsor?

Comment (Hoverbear) [reply schedule.md]: I doughnut know what you are talking about. :)

Comment (Hoverbear) [reply schedule.md]: We will have 15 minutes between vagga and Emily's talk.

I will be merging this **Saturday Morning** if there are no objections.

Comment (gnunicorn) [reply schedule.md]: Should be enough. IMHO, merge it now. Was a PR long enough @Hoverbear

Comment (Hoverbear) [reply schedule.md]: Waiting for titles from @manishearth and @edunham. :)

Comment (Manishearth) [reply schedule.md]: My title is `"Open" as in "Welcoming": Facilitating Contributions` (the subtitle can be removed if you don't find it necessary)

Still need to finish the summary, will do when back on a computer.

**Comment by Hoverbear** [comment]
@gnunicorn Can you tweet? =D

**Pull request merged**